Homework/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Tyrone Biggums</title>
+		<link rel="stylesheet" href="css/styles.css">
+		<link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css"/>
+		<link rel="stylesheet" href="http://colorzilla.com/gradient-editor/#b4ddb4+0,83c783+17,52b152+33,008a00+67,005700">
+
+	</head>
+	<body>
+
+		<nav>
+			<a href="index.html">About Me |</a>
+			<a href="portfolio.html">Portfolio</a>
+			
+		</nav>
+
+		<h1 class="text-color">SHAZAM!!!</h1>
+		<hr>
+		<h3 class="text-color">There's something you should know about me, Joe Rogan...
+			<i class="fa fa-hand-spock-o"></i>
+		</h3>
+
+		<p class="text-color">My name is Tyrone Biggums. I speak at elementary schools regarding the perils of using drugs. 
```

Comment:
@JsonLitz not saying this doesn't work, but I would probably leave the `<p>` tags specifically for content only. You have the list inside it. Something similar to achieve the same results could be something like this:
```
<div class="some-class">
  <p>All my text content lives here</p>
  <ul class="another-class">
    <li>here's my list</li>
  </ul>
</div>
```

This would visually separate the concerns and be a bit more best practice.